packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -5029,6 +5029,32 @@ export class WalletService {
     });
   }
 
+  checkServiceAvailability(req): Promise<boolean> {
+    return new Promise((resolve, reject) => {
+      if (!checkRequired(req.body, ['service', 'opts'])) {
+        return reject(new ClientError('checkServiceAvailability request missing arguments'));
+      }
+
+      let serviceEnabled: boolean;
+
+      switch (req.body.service) {
+        case '1inch':
+          if (req.body.opts && req.body.opts.country && req.body.opts.country.toUpperCase() == 'US') {
```

Comment:
This can be shortened to `req.body.opts?.country?.toUpperCase() === 'US'`

Reply:
Done. Thanks